agent/consul/federation_state_endpoint.go

```diff
@@ -11,6 +12,14 @@ import (
 	memdb "github.com/hashicorp/go-memdb"
 )
 
+const (
+	federationStatesNotEnabledMessage = "Federation states are currently disabled until all servers in the datacenter support the feature"
+)
+
+var (
+	federationStatesNotEnabled = errors.New(federationStatesNotEnabledMessage)
```

Comment:
I believe there is a convention to name error constants with an `err` (or `Err`) prefix (https://github.com/golang/go/wiki/Errors#naming), making this `errFederationStatesNotEnabled`

Reply:
Ah yeah this is from me duplicating a similar aspect of namespaces and not reevaluating the naming for what I referenced.